src/tools/clippy/.editorconfig

```diff
@@ -13,7 +13,10 @@ indent_style = space
 indent_size = 4
 
 [*.md]
+# double whitespace at end of line
+# denotes a line break in Markdown
 trim_trailing_whitespace = false
 
 [*.yml]
 indent_size = 2
+
```

Comment:
extra newline?

Reply:
Yes, to make it equal to the root `.editorconfig` and to follow best practices:

https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file
> Note that it is a good style to always put the newline as a last character if it is allowed by the file format. Furthermore, for example, for C and C++ header files it is required by the language standard.

https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

That is not important at all though, if desired, I can remove it here and in the other file. Really doesn't worth much time as far as I'm concerned.

Reply:
Yes, but line 21 already has a trailing newline, so this is an extra newline. Otherwise the GH UI would display it like this:
![image](https://user-images.githubusercontent.com/9744647/106250875-81b62d00-6214-11eb-8cb4-1e9e07107e16.png)

Reply:
@flip1995 I'm again not sure how important this is for you, but the current document in the PR has exactly one newline at the end of file:
![2021 01 29_17:23:34](https://user-images.githubusercontent.com/1547617/106300509-e003ff80-6256-11eb-8b18-5e3882e82619.png)

Or as shown in github: https://github.com/rust-lang/rust/blob/af89037f19303a6b97dac6fbab45552c44c8c97a/.editorconfig

Reply:
I mean this shouldn't be a blocker for merging this. But these are 2 newlines at the end of the file. One after line 21 and then one in line 22. See for example any other file in the repo, e.g.:
https://github.com/rust-lang/rust/blob/9fa9b58a37587d60252c6bef4b86c05027cdcd1a/rustfmt.toml#L35

This file also has a newline at the end of the file, but GitHub (and most editors) don't display an extra newline. after the last line in the document.

Reply:
Hey, thanks, I never knew editors (mousepad, nano, kate) display data this way. I confirmed with `hexdump -C`, however. I've removed the extra newline as you asked.